sql-plugin/src/main/scala/com/nvidia/spark/rapids/AstUtil.scala

```diff
@@ -32,8 +36,8 @@ object AstUtil {
    *         attributes from both join sides. In such case, it's not able
    *         to push down into single child.
    */
-  def canExtractNonAstConditionIfNeed(expr: BaseExprMeta[_], left: Seq[ExprId],
-      right: Seq[ExprId]): Boolean = {
+  def canExtractNonAstConditionIfNeed(expr: BaseExprMeta[_], left: scala.collection.Seq[ExprId],
```

Comment:
What reason are these arguments of type `scala.collection.Seq` instead of just `Seq`. Are you using mutable semantics or is the caller?

Reply:
Updated. It's not needed to update mutable in latest version.